lib/security/WSSecurity.js

```diff
@@ -3,10 +3,11 @@
 var crypto = require('crypto')
   , passwordDigest = require('../utils').passwordDigest;
 
-function WSSecurity(username, password, passwordType) {
+function WSSecurity(username, password, options) {
   this._username = username;
   this._password = password;
-  this._passwordType = passwordType || 'PasswordText';
+  this._passwordType = (options && options.passwordType) || 'PasswordText';
```

Comment:
If we "reinit" the `options` param at the very beginning, I think it could save us from checking for the `options` to be properly introduced furthermore.

```javascript
function WSSecurity(username, password, options) {
  options = options || {}; // <-- this
```